### ISSUE TITLE
convert subtitle color numbers into color codes that media players understand

### ISSUE THREAD

**Issue by Sopor-** [spaam/svtplay-dl#256]
I know that SVT use 4 colors in subtitles, but Kodi, VLC and other players don't have support for it. Is it possible to convert these numbers SVT use into color codes that players understand?
```
<36>Det otäckaste med Sävbyholm var</36>
```
36 för Ljusblå: en av personerna (oftast  huvudpersonen) i programmet. I vissa fall, t.ex. i nyhetsprogram, är det reportern som textas i ljusblått.

33 för Gul: resuméer, berättare, tal ur radio, TV, högtalare etc.

32 för Grön: sånger

35 för Lila: ljudskyltar

I don't know the color codes to use, but if you desire to add it i will find it for you if you want?

Here you have an example for yellow:
```
<font color="#ffff00">Denna texten blir gul</font>
```

**Comment by Sopor-** [comment]
https://en.wikipedia.org/wiki/ANSI_escape_code#Colors

**Comment by Sopor-** [comment]
The Retrospect addon for Kodi now have support for it in the next release.
```
    ANSIColors = {
        "<30>": '<font color="#000000">',  # Black
        "<31>": '<font color="#ff0000">',  # Red
        "<32>": '<font color="#00ff00">',  # Green
        "<33>": '<font color="#ffff00">',  # Brown/Yellow
        "<34>": '<font color="#0000ff">',  # Blue
        "<35>": '<font color="#ff00ff">',  # Magenta
        "<36>": '<font color="#00ffff">',  # Cyan
        "<37>": '<font color="#ffffff">',  # Gray
        "</30>": "</font>",
        "</31>": "</font>",
        "</32>": "</font>",
        "</33>": "</font>",
        "</34>": "</font>",
        "</35>": "</font>",
        "</36>": "</font>",
        "</37>": "</font>",
    }
```

**Comment by spaam** [comment]
i dont know. how other players handle this if they can do it or not..

**Comment by Sopor-** [comment]
Kodi, VLC, Media Player Classic and MPC-HC can handle it... Maybe you could add an option for it so it is possible to choose if you want those colors or not? If you make a GUI or a settings file it is very easy to set this option if you want colors or not.

**Comment by spaam** [comment]
@qnorsten fixed this in the pull request #406  and its merged
